py/torch_tensorrt/dynamo/partitioning/_adjacency_partitioner.py

```diff
@@ -185,6 +196,27 @@ def partition_graph(self) -> torch.fx.GraphModule:
         # Delegate nodes based on operator coverage
         subgraphs = self.put_nodes_into_subgraphs()
 
+        # A graph is fully supported if there is a single partition and all operators are supported/convertible
+        full_support = len([s for s in subgraphs if s.is_acc]) == 1 and not getattr(
+            self.operator_support, "unsupported_operators", True
+        )
+
+        if not full_support and self.require_full_compilation:
+            raise AssertionError(
+                "require_full_compilation=True was specified, but model is not fully supported"
+            )
+
+        if (
+            full_support
+            and self.require_full_compilation
+            and self.settings.min_acc_module_size != MIN_BLOCK_SIZE
```

Comment:
would the default min_acc_module_size be 0 ?

Reply:
The default would be whatever `MIN_BLOCK_SIZE` is, since that is how we specify `settings.min_acc_module_size` in the constructor. `min_acc_module_size` is effectively identical to `min_block_size`, just with a different name.

Reply:
sounds good